src/nccl_ofi_net.c

```diff
@@ -509,6 +573,20 @@ static ncclResult_t register_mr_buffers(ofiComm_t *comm, void *data,
 		ret = ncclSystemError;
 	}
 
+	if (endpoint_mr) {
+		rc = fi_mr_bind(*mr_handle, (fid_t)nccl_ofi_component[comm->dev]->ep, 0);
+		if (OFI_UNLIKELY(rc != 0)) {
+			NCCL_OFI_WARN("Unable to bind MR to EP");
+			ret = ncclSystemError;
+		}
+
+		rc = fi_mr_enable(*mr_handle);
+		if (OFI_UNLIKELY(rc != 0)) {
+			NCCL_OFI_WARN("Unable to enable EP");
```

Comment:
s/EP/memory region? Lets also add more details to error message such the dev id, mr_handle, endpoint?